### ISSUE TITLE
dylib

### ISSUE THREAD

**Issue by Venetian** [c4dm/qm-vamp-plugins#1]
should there be a .dylib file for these? (I couldn't see one)

**Comment by cannam** [comment]
Binaries aren't in the repo, they're at https://code.soundsoftware.ac.uk/projects/qm-vamp-plugins/files. (Or have I misunderstood what you're asking about?)

**Comment by Venetian** [comment]
no, I'm sure you're right! It's just the read_me was saying, drag the .dylib to the plugins folder. So I take it - easiest just to download the binaries and use this as a code repo for reference?

**Comment by cannam** [comment]
Ah, gotcha. Yes, the problem is that the README in the repo is the one that gets distributed with the binary packages, not really a README for the source code itself. I should fix that.

I also need to include some updated build instructions to accommodate the fact that it depends on qm-dsp, which isn't there in this repo and so needs to be retrieved separately. (In the original Hg repo it appears as a subrepo but the Git conversion doesn't include that.)

**Comment by Venetian** [comment]
yes, thanks. always something more ;)
If someone's able to build, they should notice the qm-dsp dependency, but a little advice on file structure, such as include or if the folders need arranging in a certain way, that could be useful.

**Comment by cannam** [comment]
I realise on second glance that I already added a note about qm-dsp, which I guess will do for now. I've made the other change though.

**Comment by Venetian** [comment]
you do work too hard!
